library/std/src/net/mod.rs

```diff
@@ -14,6 +14,34 @@
 //! * [`ToSocketAddrs`] is a trait that used for generic address resolution when interacting
 //!   with networking objects like [`TcpListener`], [`TcpStream`] or [`UdpSocket`]
 //! * Other types are return or parameter types for various methods in this module
+//!
+//! # Stability guarantees for IETF-defined behavior
+//!
+//! [`IpAddr`], [`Ipv4Addr`] and [`Ipv6Addr`] offer helper methods that provide information about
+//! an address, for instance whether it is globally routable (see [`IpAddr::is_global()`]), or if
+//! it is a multicast address (see [`IpAddr::is_multicast()`]). These methods are compliant with
+//! the [IETF RFCs]. As erratas and new RFCs are published, these methods behavior may be subject
+//! to changes.
+//!
+//! For instance, the `240/4` IPv4 block is currently reserved for "future use". If it is made
+//! globally routable by an RFC, then in a future release [`Ipv4Addr::is_reserved()`] will return
+//! `false` for such addresses, while [`Ipv4Addr::is_global`] will return true.
+//!
+//! [`IpAddr`]: ../../std/net/enum.IpAddr.html
```

Comment:
With the new rustdoc functionality of 1.48.0, these references can be written as
```
//! [`IpAddr`]: std::net::IpAddr
```

Reply:
`crate::net::IpAddr`, crates don't know their own name. Thanks for the catch :)

Reply:
I gave it a shot -- did I do it right?

Reply:
Looks great :) if you want to make the links even shorter you can use `IpAddr` instead of `crate::net::IpAddr`, I didn't realize it was in scope. Your current code will work just fine though.